cypress/integration/a11y.spec.js

```diff
@@ -80,6 +80,34 @@ describe('Check accessiblity', () => {
             findByRole('button', { name: 'Sign in' }).click();
         });
 
+        function eligibilityPages() {
+            const eligibilityPages = [
+                '1 - Is the e-Apostille accepted in the destination country?',
+                '2 - Check if the documents are eligible for the e-Apostille service',
+                '3 - Have the PDFs been notarised and digitally signed by a notary?',
+            ];
+
+            for (const [index, page] of eligibilityPages.entries()) {
```

Comment:
you can just do 
```
  eligibilityPages.forEach((page, index) => {...}) 
```

fyi, `.map`, `.filter`, `.forEach` etc will give you the item that is being iterated on (page in this case) and the index. the 3rd parameter is usually the array being iterated over `eligibilityPages` 

however I would just avoid this abstraction entirely. what was wrong with it before?

Reply:
Nothing major just a lot of repeating since the three eligibility pages need the exact same checks.

Well actually, the issue was each page only had 2 checks before and now they have 5, which was starting to look like a lot.